public/locales/en/author-guide.json

```diff
@@ -14,14 +14,13 @@
research in all areas of biology and medicine. The main features of the current eLife peer-review process are:</p><ul class=\"paragraph typography__serif typography__serif--primary\"><li>all decisions are made by editors who are active researchers</li><li>we do not artificially limit the number of articles we publish or have a set acceptance rate</li><li>editors and reviewers discuss their reviews with each other before reaching a decision on a manuscript; extra experiments, analyses, or data collection are only requested if they are essential and can be reasonably completed within about two months (please also see, “<a href=\"https://elifesciences.org/articles/57162\" target=\"_blank\">Publishing in the time of COVID-19</a>”)</li><li>the public reviews, revision recommendations, and author responses are published with accepted manuscripts</li></ul><p class=\"paragraph typography__serif typography__serif--primary\">The overall aim is to make peer review faster (by reducing rounds of revision and only requesting extra experiments when they are essential), fairer and more open. Around 30% of submitted manuscripts are sent to external experts for in-depth peer review, and our overall acceptance rate is around 16%.</p><p class=\"paragraph typography__serif typography__serif--primary\">We are committed to treating all authors and manuscripts fairly, and we offer \"scoop protection\" in the sense that, if other researchers publish similar findings after submission, this will not be a reason for rejection. Further details are in our <a href=\"https://submit.elifesciences.org/html/elife_author_instructions.html#faq\" target=\"_blank\">FAQs</a>.</p><p class=\"typography__serif typography__serif--primary\">Authors can <a href=\"https://reviewer.elifesciences.org/login\" target=\"_blank\">submit to eLife directly</a>, they can submit using the <a href=\"https://www.overleaf.com/latex/templates/elife-latex-template/csqxykvsnyxm#.WIs5NLaLRGw\" target=\"_blank\">Overleaf</a> or <a href=\"https://www.authorea.com/\" target=\"_blank\">Authorea</a> authorship tools, or they can transfer their files from <a href=\"http://biorxiv.org/\" target=\"_blank\">bioRxiv</a> or <a href=\"https://www.medrxiv.org/\" target=\"_blank\">medRxiv</a> if they have already posted a preprint. Authors working in LaTeX can download and use our <a href=\"https://elife-cdn.s3.amazonaws.com/author-guide/elife-latex-template.zip\" target=\"_blank\">template</a> or <a href=\"https://www.overleaf.com/latex/templates/elife-latex-template/csqxykvsnyxm\" target=\"_blank\">open it directly in Overleaf</a>.</p><p class=\"paragraph typography__serif typography__serif--strong\">Please note: eLife’s publication fee will increase to $3,000 for authors submitting on or after April 5, 2021 (<aPlease note: eLife’s publication fee will increase to $3,000 for authors submitting on or after April 5, 2021 (<a href=\"https://submit.elifesciences.org/html/elife_author_instructions.html\">read more</a>).</p>",
```

Comment:
Looks like a copy/paste issue at the end of the sentence. I can see `(<aP` there is an anchor tag that isn't closed properly and I can see the text `Please note: eLife’s publication fee will increase to $3,000 for authors submitting on or after April 5, 2021` twice.